code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1190,3 +1190,47 @@
   */
 /mob/living/carbon/proc/get_biological_state()
 	return BIO_FLESH_BONE
+
+/// Returns maximum number of active chip skills for this mob
+/mob/living/carbon/proc/get_max_skillchip_slot_count()
+	. = 2
+	if(dna && dna.check_mutation(BIOTECHCOMPAT))
+		. += 1
+
+/// Returns sum of slots used by active chips
+/mob/living/carbon/proc/get_used_skillchip_slot_count()
+	. = 0
+	var/obj/item/organ/brain/B = getorganslot(ORGAN_SLOT_BRAIN)
+	if(!B)
+		return
+	for(var/obj/item/skillchip/S in B)
+		. += S.slot_cost
```

Comment:
Same here, a carbon variable that gets increased and decreased by skillchips would be much better long-term.
Then we can easily react to the change events.